projects/demo/src/modules/components/combo-box/examples/1/index.ts

```diff
@@ -44,16 +45,20 @@ export class TuiComboBoxExample1 {
 
     readonly testValue = new FormControl(databaseMockData[1]);
 
-    onSearchChange(searchQuery: string) {
+    onSearchChange(searchQuery: string | null) {
         this.search$.next(searchQuery);
     }
 
+    extractValueFromEvent(event: Event): string {
+        return (event.target as EventTarget & {value?: string})?.value ?? '';
+    }
+
     /**
      * Service request emulation
      */
-    private serverRequest(searchQuery: string): Observable<readonly User[]> {
+    private serverRequest(searchQuery: string | null): Observable<readonly User[]> {
         const result = databaseMockData.filter(user =>
-            user.toString().toLowerCase().includes(searchQuery.toLowerCase()),
+            searchQuery ? TUI_DEFAULT_MATCHER(user, searchQuery) : user,
```

Comment:
Maybe just `TUI_DEFAULT_MATCHER(user, searchQuery || '')` will do the same?

Reply:
fixed